lab2/README.md

```diff
@@ -0,0 +1,58 @@
+# Lab 2
+
+Your goal for this lab is to make an app that uses two APIs, where one of them has to be an Android API. 
+
+You will have 2 weeks to complete this lab. **Lab 2 is due at 6:30pm on Thursday, October 27th.** This lab will be done in groups of either two or three. There should be two groups of three. You should work with new people on this lab.
+
+[Here](./RUBRIC.md) is the rubric.
+
+## Deliverables
+We require these two things to be turned in:
+* Your proposal + Wireframes + Design Explanation (Sun Oct 16)
+* Your app (Thu Oct 27)
+
+## MVP
+The only requirement for this lab is that you use two APIs. One API has to be an Android API while the other one can be either an Android or web API. All other requirements are scoped by your team in your proposal.
```

Comment:
This is a little confusing just because it's not clear what an "Android API" is. For example, Amazon s3 has an Android SDK, but I don't think you would count that as an Android API. Maybe you mean an Andorid API that integrates with your phones' hardware?